tests/functional/views/test_legacy_publishing.py

```diff
@@ -365,7 +362,7 @@ def test_publishing_overwrite_collection_litezip(
     )
     # FIXME: uncomment.
     # assert resp.status_code == 204
-    assert resp.status_code == 400  # no changes to collection.xml
+    assert resp.status_code == 200
```

Comment:
Hmm... was this previously 204 and it is now 200?

Reply:
Yeah.. I'm not sure what this means